src/test/java/org/openstreetmap/atlas/geography/atlas/items/complex/restriction/ComplexTurnRestrictionTest.java

```diff
@@ -191,6 +191,23 @@ public void testTurnRestrictionNoUTurn()
         Assert.assertTrue(possibleTurnRestriction.isPresent());
     }
 
+    @Test
+    public void testTurnRestrictionTags()
+    {
+        final Atlas testAtlas = this.rule.getAtlasNoUTurn();
+
+        final Optional<TurnRestriction> possibleTurnRestriction = TurnRestriction
+                .from(testAtlas.relation(1L));
+
+        Assert.assertTrue(possibleTurnRestriction.isPresent());
+
+        final TurnRestriction turnRestriction = possibleTurnRestriction.get();
+
+        // Make sure both tags exist
+        Assert.assertEquals(2, turnRestriction.getTags().size());
+        Assert.assertEquals("no_left_turn", turnRestriction.getTag("restriction").get());
```

Comment:
Maybe use `TurnRestrictionTag` here instead of strings?

Reply:
Oh - great find! Yes - I will update this to use the TurnRestrictionTag statics!